examples/CLI/ReadMe.md

```diff
@@ -0,0 +1,17 @@
+# Quickstart - AzureML CLI Model Analysis
```

Comment:
We shouldn't use "Model analysis" -- instead "Azure ML CLI Responsible AI Insights"

Reply:
Done